agent/api/task_unix.go

```diff
@@ -15,16 +15,126 @@
 
 package api
 
+import (
+	"path/filepath"
+
+	"github.com/aws/amazon-ecs-agent/agent/config"
+	"github.com/aws/amazon-ecs-agent/agent/utils"
+	"github.com/cihub/seelog"
+	docker "github.com/fsouza/go-dockerclient"
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
+)
+
 const (
 	portBindingHostIP = "0.0.0.0"
+	maxTaskVCPULimit  = 10     // 10 VCPUs limit
```

Comment:
please add some documentation here, especially about what it means to set this to 10 VCPUs.

Reply:
Sure